### ISSUE TITLE
Fail to create IBC Channel in EVM Rollapp

### ISSUE THREAD

**Issue by Adamtruong6868** [dymensionxyz/roller#1043]
### Roller version

1.6.4-alpha-rc08

### Rollpp type (EVM/WASM) and version

EVM

### OS

Linux

### Processor Architecture

AMD 8 cores

### Others

16G RAM, 400GB

### Steps to reproduce it

roller relayer setup

### Expected result

Creating eIBC channel successfully

### Actual result

Fail to create IBC channel

### Log Files

[relayer.log](https://github.com/user-attachments/files/17146715/relayer.log)
[rollapp.log](https://github.com/user-attachments/files/17146716/rollapp.log)

### Notes

Randomly happen on EVM rollapps.

**Comment by barburan1** [comment]
i have same mistake, 
![image](https://github.com/user-attachments/assets/f0519c29-53d4-46fc-9bf3-634c34e33cac)
failed to create IBC channel: command execution failed: exit status 1

**Comment by danwt** [comment]
```
ight provider light block: client provided bad signed header: expected validator hash of header to match validator set hash (A130487864D20F55E7AB5876931AF389088B94E29D74EBE258B75D54CFE7493A != ADC9E5E48253EE56F29FA95327C3CDB5B7DC5FC54E4DF722E97DF66D7E066F37)

```

**Comment by artemijspavlovs** [comment]
was addressed in the latest `roller` release `v1.7.3-alpha*`
